spec/commands/thredded/moderate_post_spec.rb

```diff
@@ -38,5 +38,27 @@ module Thredded
       expect { Thredded::ModeratePost.run!(post: topic.first_post, moderation_state: :blocked, moderator: moderator) }
         .to_not change { topic.first_post.reload.updated_at }
     end
+
+    context 'when not Thredded.content_visible_while_pending_moderation' do
+      around { |ex| with_thredded_setting(:content_visible_while_pending_moderation, false, &ex) }
```

Comment:
What do you think about expanding `ex` to `example` and breaking this out to multi-line do/end block. Took me a little bit to parse this out.